packages/yourloops/components/patient/add-dialog.tsx

```diff
@@ -199,7 +199,17 @@ function AddDialog(props: AddDialogProps): JSX.Element {
               value={teamId}
               onChange={handleChangeTeam}
             >
-              {optionsTeamsElements}
+              <MenuItem id="patient-list-dialog-add-team-option-none" aria-label={t('none')} value="" key="none" />
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is this long ass id really necessary? I don't see it being used

And what does `key="none"` do?